src/audiences/edit/index.js

```diff
@@ -0,0 +1,235 @@
+												<option value="!=">{ __( 'does not equal', 'altis-analytics' ) }</option>
+												<option value="*=">{ __( 'contains', 'altis-analytics' ) }</option>
+												<option value="!*">{ __( 'does not contain', 'altis-analytics' ) }</option>
+												<option value="^=">{ __( 'begins with', 'altis-analytics' ) }</option>
+											</Fragment>
+										) }
+										{ currentField.type === 'number' && (
+											<Fragment>
+												<option value="gt">{ __( 'is greater than', 'altis-analytics' ) }</option>
+												<option value="lt">{ __( 'is less than', 'altis-analytics' ) }</option>
+												<option value="gte">{ __( 'is greater than or equal to', 'altis-analytics' ) }</option>
+												<option value="lte">{ __( 'is less than or equal to', 'altis-analytics' ) }</option>
+											</Fragment>
+										) }
+									</select>
+									{ ! currentField.type || currentField.type === 'string' && (
+										<Fragment>
+											{ [ '=', '!=' ].indexOf( rule.operator ) >= 0 && (
+												<select
+													className="audience-editor__rule-value"
+													onChange={ e => updateRule( groupId, ruleId, { value: e.target.value } ) }
+													value={ rule.value }
+													name={`audience[groups][${groupId}][rules][${ruleId}][value]`}
+												>
+													<option value="">{ __( 'Empty', 'altis-analytics' ) }</option>
+													{ fields.filter( field => field.name === rule.field ).map( field => field.data && field.data.map( ( datum, index ) => (
+														<option key={ index } value={ datum.value }>{ datum.value }</option>
+													) ) ) }
+												</select>
+											) }
+											{ [ '*=', '!*', '^=' ].indexOf( rule.operator ) >= 0 && (
+												<input
+													className="regular-text"
+													type="text"
+													value={ rule.value }
+													onChange={ e => updateRule( groupId, ruleId, { value: e.target.value } ) }
+													name={`audience[groups][${groupId}][rules][${ruleId}][value]`}
+												/>
+											) }
+										</Fragment>
+									) }
+									{ currentField.type === 'number' && (
+										<input
+											className="regular-text"
+											type="number"
+											placeholder={ `${ __( 'Average value: ', 'altis-analytics' ) } ${ currentField.stats.avg }` }
+											value={ rule.value }
+											onChange={ e => updateRule( groupId, ruleId, { value: e.target.value } ) }
+											name={`audience[groups][${groupId}][rules][${ruleId}][value]`}
+										/>
+									) }
+
+									{ group.rules.length > 1 && (
+										<Button
+											className="audience-editor__rule-remove"
+											isDestructive={ true }
+											isLink={ true }
+											onClick={ () => {
+												const newRules = group.rules.slice();
+												newRules.splice( ruleId, 1 );
+												updateGroup( groupId, { rules: newRules } );
+											} }
```

Comment:
For readability, it would be good for this to be factored out as a `onRemove` function rather than being defined inline.